src/dibbs_text_to_code/services/extraction.py

```diff
@@ -0,0 +1,40 @@
+from dibbs_text_to_code import configs
+
```

Comment:
I do think you should add at least one test to show we've returned a tensor object that is the same dimensions as the model expects (768 for Qwen). I could be wrong, but I don't think you'll have to mock anything as long as you load this in the test file `from embeddings import embed`

Reply:
I like this idea as well!